hardware/drone/camera/camera_controller.py

```
@@ -0,0 +1,151 @@
+        if not self.is_streaming or not self.tello:
+            return
+
+        try:
+            frame = self.tello.get_frame_read().frame
+            
+            if frame is not None:
+                # Convert BGR to RGB
+                frame_rgb = cv2.cvtColor(frame, cv2.COLOR_BGR2RGB)
+                
+                # Convert to QImage
+                height, width, channel = frame_rgb.shape
```

Ruff (0.14.2)

Unpacked variable `channel` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)